<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="nazov_suboru">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nazov:</string>
     </property>
    </widget>
    <widget class="QLabel" name="typ_suboru">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Typ:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sub_velkost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Velkost:</string>
     </property>
    </widget>
    <widget class="QLabel" name="vytvorenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vytvorene:</string>
     </property>
    </widget>
    <widget class="QLabel" name="suborPosledna">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Posledna uprava:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cesta_suboru">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Path: </string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Informacie o súbore</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Privatny kluc:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="private_key_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Path: </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_sign">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Overenie Podpisu</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="public_key_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Path: </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_check_sign">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Sign</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Verejný klúč</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tvorba Klučov</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_genKeys">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vytvorenie Podpisu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAfinn_ifra">
    <property name="title">
     <string>Elektronický Podpis</string>
    </property>
   </widget>
   <addaction name="menuAfinn_ifra"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
